dts/bindings/i2c/i2c-device.yaml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Add license header

Reply:
ok, it seems it's missing in bunch of them.
Will do something

Reply:
@MaureenHelm: What copyright should be provided for files that I didn't created?
Should I look for the original author and put his copyright?
Or can I just put SPDX boiler plate w/o any copyright?

Reply:
Use your own copyright for the new files you add